cfgov/v1/models/molecules.py

```diff
@@ -31,6 +33,27 @@ class ImageText5050(blocks.StructBlock):
     link_url = blocks.URLBlock(required=False)
     link_text = blocks.CharBlock(max_length=100, required=False)
 
+    def clean(self, data):
+        ordered_tuples = data.items()
```

Comment:
For more info, see [this](https://docs.djangoproject.com/en/1.8/ref/forms/validation/#cleaning-and-validating-fields-that-depend-on-each-other).